examples/deltaFunctionKernels.cc

```diff
@@ -23,7 +23,7 @@
 #include <iostream>
 #include <vector>
 
-#include "boost/shared_ptr.hpp"
+#include <memory>
```

Comment:
please move up one line (blank line after std imports); I know it's picky, but...